app/com/arpnetworking/metrics/portal/alerts/impl/PluggableAlertRepository.java

```diff
@@ -102,13 +108,16 @@ public PluggableAlertRepository(
             final PeriodicMetrics periodicMetrics,
             final Injector injector,
             final Environment environment,
+            @Named("AlertJobCoordinator")
+            final ActorRef alertJobCoordinator,
```

Comment:
Should this then be marked `@Nullable`? 

Question: Why is it not always provided? Is this because it's cluster singleton and only runs on one node? If so, can you add that to the construct description.

Reply:
That was really just for testing. The constructor tied into Guice will always provide the actor ref.

Reply:
I made this non-null / non-optional to avoid confusion. It was only empty in one test case anyway.